forge/ee/db/models/Subscription.js

```diff
@@ -63,6 +63,7 @@ module.exports = {
     },
     associations: function (M) {
         this.belongsTo(M.Team)
+        M.Team.hasOne(this)
```

Comment:
This is needed so sequelize knows about the reverse association from Team to Subscription - allowing us to 'include' the Subscription model when querying Teams. As we no long use the `sync()` function to create the tables, this has no impact on the table structure.